docs/admin/deployment/onboarding_journalists.rst

```diff
@@ -103,9 +104,15 @@ to access the servers over SSH.
     cd ~/Persistent/securedrop
     ./securedrop-admin setup
     ./securedrop-admin tailsconfig
+    rm install_files/ansible-base/group_vars/all/site-specific
 
   .. note:: The ``setup`` command may take several minutes, and may fail partway
             due to network issues. If so, run it again before proceeding.
+            
+            The ``site-specific`` file should be removed after running the
```

Comment:
From a technical perspective this PR is fine. But I'm slightly concerned about this because the JI is internet connected and well, if it's sensitive, we shouldn't be copying it there.

The main mitigating factor is that this is a one-time setup thing and the file should be deleted before the JW ever gets in the hands of a journalist. Maybe we can discuss this briefly during tomorrow's stand up or (since it'll be Wednesday) team meeting?